services/api/handlers/projects.js

```diff
@@ -49,6 +49,25 @@ exports.post = {
 };
 
 exports.get = {
+  oneById: function(request, reply) {
+    request.server.methods.projects.findOneById(
```

Comment:
for ref: 
`findOneById`: https://github.com/mozilla/api.webmaker.org/blob/develop/services/api/lib/queries.js#L113-L114